### PR TITLE
completed typescript-objects - Carolyne Golias

### PR DESCRIPTION
**Pull request by GCarolyne**
Screenshot 2024-10-22 at 12 09 04 PM

### CODE REVIEW

File: typescript-objects/main.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+interface StudentProps {
+  firstName: string;
+  lastName: string;
+  age: number;
+  livesInIrvine?: boolean;
+  previousOccupation?: string;
+}
+const student: StudentProps = {
+  firstName: ' Carolyne ',
+  lastName: ' Golias ',
```
Comment (shawnkost) [inline typescript-objects/main.ts]: You should never really store spaces in strings like this. If you need to add spaces when logging things, you would add the space in the concatenation

File: typescript-objects/main.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+vehicle['color'] = 'blue';
+vehicle['isConvertible'] = false;
+console.log('The color of the vehicle', vehicle['color']);
+console.log('Is it a convertible', vehicle['isConvertible']);
+console.log('The value of the vehicle object', vehicle);
+console.log('The type of vehicle', typeof vehicle);
+
+interface Pet {
+  name: string;
+  kind: string;
```
Comment (shawnkost) [inline typescript-objects/main.ts]: These should be marked as optional because they get deleted

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Carolyne! Just a couple small changes here

File: typescript-objects/main.ts (hunk 1 above)
Comment (GCarolyne) [reply typescript-objects/main.ts]: I understand. Thank you!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**